src/cmds/scripts/win_postinstall.py

```diff
@@ -161,17 +162,50 @@ def install_vcredist():
 def create_pbs_conf():
     __log_info('Creating PBSPro configuration at %s' % pbs_conf_path)
     with open(pbs_conf_path, 'w+') as fp:
-        pbs_conf_data = pbs_conf_t.substitute(pbs_server=gethostname(),
+        if installtype == 'server':
+            pbs_conf_data = pbs_conf_t.substitute(pbs_server=gethostname(),
                                               pbs_exec=pbs_exec,
-                                              pbs_home=pbs_home)
+                                              pbs_home=pbs_home,
+                                              server_flag=1,
+                                              comm_flag=1,
+                                              mom_flag=1,
+                                              sched_flag=1)
+        elif installtype == 'execution':
+            pbs_conf_data = pbs_conf_t.substitute(pbs_server='CHANGE_THIS_TO_PBS_PRO_SERVER_HOSTNAME',
+                                              pbs_exec=pbs_exec,
+                                              pbs_home=pbs_home,
+                                              server_flag=0,
+                                              comm_flag=0,
+                                              mom_flag=1,
+                                              sched_flag=0)
+        elif installtype == 'client':
+            pbs_conf_data = pbs_conf_t.substitute(pbs_server='CHANGE_THIS_TO_PBS_PRO_SERVER_HOSTNAME',
+                                              pbs_exec=pbs_exec,
+                                              pbs_home=pbs_home,
+                                              server_flag=0,
+                                              comm_flag=0,
+                                              sched_flag=0,
+                                              mom_flag=0)
+        elif installtype == 'comm':
+            pbs_conf_data = pbs_conf_t.substitute(pbs_server='CHANGE_THIS_TO_PBS_PRO_SERVER_HOSTNAME',
+                                              pbs_exec=pbs_exec,
+                                              pbs_home=pbs_home,
+                                              server_flag=0,
+                                              sched_flag=0,
+                                              mom_flag=0,
+                                              comm_flag=1)
         fp.write(pbs_conf_data.lstrip())
     __log_info('Successfully created PBSPro configuration')
 
 
 def create_home():
     __log_info('Creating PBSPro home')
     # No need to set correct permission here as it will be done by pbs_mkdirs
-    os.makedirs(pbs_home)
+    try:
+        os.makedirs(pbs_home)
+    except WindowsError:
+        __log_info('PBS_HOME directory is already available, delete PBS_HOME and execute this script again')
```

Comment:
Yes, Its needed only while executing this script second time due to to failure for network misconfiguration. We wanted to print a nice message instead of throwing and exception to user. Its purely "nice to have" not a functional change.